docs/userGuide/contentAuthoring.md

```diff
@@ -143,9 +143,15 @@ Note: Replace all instances of `-` with `_` in the glyph's name, e.g. <code>{<sp
 
 #### Timestamp
 
-Use <code>{<span></span>{timestamp}}</code> to insert the following snippet that indicates when the page was generated.  
+Use <code>{<span></span>{timestamp}}</code> to insert a UTC time snippet that indicates when the page was generated.  
 
-{{timestamp}}
+<code>{<span></span>{timestamp}}</code> will be rendered as: {{timestamp}}
+
+#### MarkBind Link
+
+Reference or support MarkBind by using <code>{<span></span>{MarkBind}}</code>, which will insert a link back to our website.

```

Comment:
- Let's use the same manner of speech "`to insert`" instead of "`, which will insert`".
- "`back`" is unnecessary.